features/admin/deactivate_supplier_user.feature

```diff
@@ -0,0 +1,41 @@
+@deactivate-supplier-user
+Feature: Deactivate a supplier's contributor
+
+Scenario Outline: Correct users can deactivate a supplier's contributor
```

Comment:
Really minor but should this be `Correct users can deactivate and reactivate a supplier's contributor`?

Reply:
Done